spec/providers/coreIntegration/udpsocket.integration.src.js

```diff
@@ -8,71 +8,70 @@ module.exports = function(provider, setup) {
   });
 
   it("Connects, has state, and sends/receives data", function (done) {
-    // Currently a copy of 'receives data' unit test from firefox, change it
+    const todo = [];  // Pending tasks to complete before calling |done()|.
```

Comment:
I'd prefer the consistency of keeping everything as ```var```, and it
shouldn't break anything.
It's a feature where I still don't feel like I fully understand the
implications of calling soemthing const, and I'd like to see it enter
common parlance a bit more before adopting it.

On Fri, Jan 2, 2015 at 7:57 PM, Benjamin M. Schwartz <
notifications@github.com> wrote:

> In spec/providers/coreIntegration/udpsocket.integration.src.js
> <https://github.com/freedomjs/freedom/pull/182#discussion-diff-22418694>:
>
> > @@ -8,71 +8,70 @@ module.exports = function(provider, setup) {
> >    });
> >
> >    it("Connects, has state, and sends/receives data", function (done) {
> > -    // Currently a copy of 'receives data' unit test from firefox, change it
> > +    const todo = [];  // Pending tasks to complete before calling |done()|.
>
> As you may have noticed, this test used const exclusively. Would you like
> me to change them all?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/freedomjs/freedom/pull/182/files#r22418694>.
>